core-bundle/src/Util/PackageUtil.php

```diff
@@ -60,7 +59,7 @@ public static function parseVersion(string $version): string
     {
         trigger_deprecation('contao/core-bundle', '4.13', 'Using the PackageUtil::parseVersion() method has been deprecated and will no longer work in Contao 5.0. Use the Composer\InstalledVersions class instead.');
 
-        return ltrim(strstr($version, '@', true), 'v');
+        return ltrim(strstr($version.'@', '@', true), 'v');
```

Comment:
```suggestion
        return ltrim($version, 'v');
```

We could simplify this, because `InstalledVersions::getPrettyVersion()` will never return the `@<hash>` part.

Reply:
But the method is public so it has to support inputs that previously worked.

Reply:
You are right of course. 🦊